### ISSUE TITLE
Doriesit login nastavenia pre eid

### ISSUE THREAD

**Issue by jsuchal** [slovensko-digital/navody.digital#668]
callbacky teraz nedavaju uplne zmysel, chcelo by to 2 komponenty. Avsak realny zmysel dava vlastne normalna sso integracia a potom pouzitie cez podaas (backend), tak ako to vsetkym odporucame.

V NASES by nam mohli uznat noveho service providera pod podaas na navody.digital nie? @luciajanikova

**Comment by luciajanikova** [comment]
Ano, registracia noveho SP priamo pre navody je to iste, ako ked pridame dalsi komponent (co uz aj mame a problem nie je).

**Comment by jsuchal** [comment]
@luciajanikova to by som teda regol (aj staging aj prod) a nechame to na nejaky hackday dalsi alebo fixneme my
